app/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -50,6 +52,7 @@ class EmailVerificationConnectorImpl @Inject() (http: WSHttp, metrics: Metrics)(
 
   def verifyEmail(nino:           String,
                   newEmail:       String,
+                  name:           Name,
                   isNewApplicant: Boolean)(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[Either[VerifyEmailError, Unit]] = {
```

Comment:
why don't we add all these params to a case class ?

Reply:
depends if we want a case class especially for this method - it wouldn't be used elsewhere so it's re-usability is limited. I'll give it a a go and see what it looks like